docs/.vuepress/tools/check-links.js

```diff
@@ -0,0 +1,146 @@
+const { SiteChecker } = require('broken-link-checker'); // eslint-disable-line import/no-unresolved
+const path = require('path');
+const execa = require('execa');
+const { Renderer } = require('xlsx-renderer');
+const { logger } = require('./utils');
+
+const ACCEPTABLE_STATUS_CODES = [undefined, 200, 429];
+const PORT = 8088;
+
+const spawnProcess = (command, options = {}) => {
+  const cmdSplit = command.split(' ');
+  const mainCmd = cmdSplit[0];
+
+  cmdSplit.shift();
+
+  if (!options.silent) {
+    options.stdin = options.stdin ?? 'inherit';
+    options.stdout = options.stdout ?? 'inherit';
+    options.stderr = options.stderr ?? 'inherit';
+  }
+
+  return execa(mainCmd, cmdSplit, options);
+};
+
+// start server
+spawnProcess(`http-server ${path.resolve('.vuepress', 'dist')} -s -p ${PORT}`);
+const stats = {
+  brokenInternal: 0,
+  brokenExternal: 0,
+  external: 0,
+  internal: 0,
+  _set: new Set()
+};
+const links = [];
+
+const saveReport = async() => {
+  const result = await new Renderer().renderFromFile(path.resolve(__dirname, './check-links-report-template.xlsx'), {
```

Comment:
Do we really need another dependency and produce XLSX file to display broken links? Simple txt file with "what" and "where" can be sufficient to fix those. Or reported directly by the linter

Reply:
1. It contains much more information than only broken links, It contains all links between whole pages. With a filter feature in Excel, it is easy to use tool for working with our documentation.
2. It is easy to use, filter, and analyze out of a box. Do not require importing data each I want to analyze links. I'm using the report much time and it is awesome - I love this solution ❤️ 
3. It is only dev dependency

Reply:
How often are you going to use it after this project is finished? Any dev dependency is additional installation time, payload and waiting for our devs. May clog CI time as well. I get it, it's useful to have those links in spreadsheet. But very same could be achieved by extracting to CSV by joining strings with a comma.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik